R/alt_aggregate_counts.R

```diff
@@ -96,12 +95,13 @@ alt_aggregate_counts <- function(
 
         pri_df <- state_df %>%
             filter(Web.Group == "Prison") %>%
-            full_join(mp_df, by = c("State", "Date", "Measure"))
+            full_join(mp_df, by = c("State", "Date", "Measure")) %>% 
+            mutate(Web.Group = "Prison")
```

Comment:
We need to assign the `Web.Group` for the rows we get from MP to `Prison` (otherwise the `Web.Group` is NA and they get dropped). This meant we were losing data for places like Wyoming where we only have MP data for `Residents.Confirmed`.